test/terraform_ssh_example_test.go

```diff
@@ -152,7 +152,9 @@ func testSSHToPublicHost(t *testing.T, terraformOptions *terraform.Options, keyP
 func testSSHToPrivateHost(t *testing.T, terraformOptions *terraform.Options, keyPair *aws.Ec2Keypair) {
 	// Run `terraform output` to get the value of an output variable
 	publicInstanceIP := terraform.Output(t, terraformOptions, "public_instance_ip")
-	privateInstanceIP := terraform.Output(t, terraformOptions, "private_instance_ip")
+
+	// Get IP of private instance from AWS helper function instead of Terraform output
+	privateInstanceIP := aws.GetPrivateIpOfEc2Instance(t, terraform.Output(t, terraformOptions, "private_instance_id"), terraformOptions.Vars["aws_region"].(string))
```

Comment:
Could you break this up into a few lines? One to read the `instanceId`, one to read the `awsRegion`, and then one to get the `privateInstanceIP` using the previous two.

Reply:
Sure